pkg/kubelet/rkt/container_id.go

```diff
@@ -41,6 +42,14 @@ func buildContainerID(c *containerID) kubecontainer.ContainerID {
 	}
 }
 
+func apiPodToNetPluginContainerID(pod *api.Pod) kubecontainer.ContainerID {
+	return buildContainerID(&containerID{string(pod.UID), pod.Name})
```

Comment:
See the implementation of `buildContainerID` above, it always, at the least, injects a `:` we don't want.

I did this via `kubecontainer.ContainerID{ID: string(pod.UID)})` to create a kubecontainer ID that worked correctly with `SetUpPod`